BackofficeBundle/Form/DataTransformer/ChoicesOptionToArrayTransformer.php

```diff
@@ -56,4 +46,27 @@ public function reverseTransform($choices)
 
         return $arrayChoice;
     }
+
+    /**
+     * @param string $choice
+     *
+     * @return string
+     */
+    protected function suppressSpecialCharacterTranslateChoice($choice)
```

Comment:
Because they are not the same, the SuppressSpecialCharacterTransformer doesn't allow the character '_' and '.'